Sources/TSCBasic/FileSystem.swift

```diff
@@ -760,6 +777,21 @@ public class InMemoryFileSystem: FileSystem {
 
         contents.entries[sourcePath.basename] = nil
     }
+
+    public func withLock<T>(on path: AbsolutePath, type: LockType = .exclusive, _ body: () throws -> T) throws -> T {
+        var fileQueue: DispatchQueue
+
+        lockFilesLock.lock()
+        if let queue = lockFiles[path] {
+            fileQueue = queue
+        } else {
+            fileQueue = DispatchQueue(label: "foo", attributes: .concurrent)
```

Comment:
foo?

Reply:
I forgot to remove the placeholder. This should be org.swift.swiftpm.in-memory-file-system.file-queue